articles/resources/relnotes.md

```diff
@@ -17,6 +17,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.15.2101.126807
+
+*Release date: January 29th, 2021*
+
+- Added project templates to Q# compiler for executables targeting IonQ and Honeywell providers
+- Update IQ# kernel syntax highlighting to include changes to Q# syntax introduced in version 0.15.2101125897
+- Bugfix to support passing arrays as input arguments to Q# programs submitted to Azure Quantum via `%azure.execute`
```

Comment:
Can we link to https://github.com/microsoft/iqsharp/issues/401 here?

Reply:
Done! Thanks